docs/modules/ROOT/pages/custom-attributes.adoc

```diff
@@ -0,0 +1,99 @@
+= Option: Custom attributes string
+:navtitle: Custom attributes string
+
+The extension option *Custom attributes string* is a space-separated list of attribute settings.
+
+You may use the above string, and other methods to set attributes that affect the document preview.
+For all methods and when they are applied, visit the xref:features.adoc#set-attributes[Set attributes] section on the Features page.
+
+== Use
+
+The *Custom attributes string* option allows you to use the features of AsciiDoc in documents without having to write them each time, or include a separate file of attribute settings.
+
+For example, to use the GUI macros for every AsciiDoc you preview with the extension, you can set  `experimental` in the *Custom attributes string*.
+
+[,txt]
+----
+experimental
+----
+
+== AsciiDoc Attributes
+
+To learn more about AsciiDoc attributes, visit the
+xref:asciidoc:attributes:document-attributes.adoc[]
+page of the AsciiDoc manual.
+
+== Setting attribute values
+
+Attributes can be set with and without a value, and unset.
+The following table shows examples of setting attributes in AsciiDoc with their Custom Attribute equivalents.
+
+[cols="3*a"]
+|===
+|Action |AsciiDoc |Custom Attributes example
+
+|Set without a value |`:sectnums:` |`sectnums` 
+|Set to a value |`:toc: left` |`toc=left` 
+|Unset |`:sectnums!:` |`sectnums!`
+
+|===
+
+NOTE: Setting an attribute in the string is not a guarantee that it will take effect.
+See the xref:features.adoc#se
```

Comment:
The xref is incomplete, it must end with `[]`

Reply:
Correct reference added.